rllib/evaluation/rollout_worker.py

```diff
@@ -2045,7 +2038,11 @@ def _get_complete_policy_specs_dict(
                 # Policies should deal with preprocessed (automatically flattened)
                 # observations if preprocessing is enabled.
                 preprocessor = ModelCatalog.get_preprocessor_for_space(
-                    obs_space, merged_conf.model
+                    obs_space,
+                    merged_conf.model,
+                    include_multi_binary=self.config.get(
+                        "_enable_rl_module_api", False
+                    ),
```

Comment:
This is our only entry point to ModelCatalog.get_preprocessor_for_space.
This extra argument is not a pretty solution, but by far the best one I could think of for now.
This PR includes comments on inlining the preprocessor method, which is when we'll be able to get rid of this method.